test/unittest/core_svc_test.yaml

```diff
@@ -0,0 +1,84 @@
+suite: CoreSvc
+
+tests:
+  - it: Annotation
+    set:
+      core:
+        serviceAnnotations:
+          test.annotation: test-annotation
+    template: templates/core/core-svc.yaml
+    asserts:
+      - equal:
+          path: metadata.annotations["test.annotation"]
+          value: test-annotation
+
+  - it: TypeGce
+    set:
+      expose:
+        ingress:
+          controller: gce
+    template: templates/core/core-svc.yaml
+    asserts:
+      - equal:
+          path: spec.type
+          value: NodePort
+
+  - it: TypeAlb
+    set:
+      expose:
+        ingress:
+          controller: alb
+    template: templates/core/core-svc.yaml
+    asserts:
+      - equal:
+          path: spec.type
+          value: NodePort
+
+  - it: TypeF5
+    set:
+      expose:
+        ingress:
+          controller: f5-bigip
+    template: templates/core/core-svc.yaml
+    asserts:
+      - equal:
+          path: spec.type
+          value: NodePort
+
+  - it: NotType
+    template: templates/core/core-svc.yaml
+    asserts:
+      - notExists:
+          path: spec.type
+
+  - it: ExposeTLSEnabled
```

Comment:
Since `ExposeTLSEnabled` can be misunderstood as `expose.tls.enabled`, maybe change it to `InternalTLSEnabled`?